### ISSUE TITLE
New Proposal - Registry Edit

### ISSUE THREAD

**Issue by sebuhbitcoin** [dOrgTech/homebase-app#202]
[
  {
    "kind": "temporary",
    "id": "proto.009-PsFLoren.michelson_v1.runtime_error",
    "contract_handle": "KT1XAtu9pqFFbnjbppB6JYos8ZxtawwJZfVF"
  },
  {
    "kind": "temporary",
    "id": "proto.009-PsFLoren.michelson_v1.script_rejected",
    "location": 2310,
    "with": {
      "string": "FAIL_PROPOSAL_CHECK"
    }
  }
]

I get this error when trying to make a new Proposal.

![image](https://user-images.githubusercontent.com/5566620/122690133-93589b00-d1dc-11eb-9674-740af15de782.png)

Basically I want to make a Vote for the DAO that goes like this.

What should Sebuh.net Vote for on Granda Promtion Phase?
Yay , or Nay,

And present the users 2 options,
to vote yay or nay.

Thanks.
